src/cpp/continuous_batching/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ find_file(spda_to_pa_header sdpa_to_paged_attention.hpp
 
 # end of dependencies
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
you can set it as target property 
https://cmake.org/cmake/help/latest/prop_tgt/POSITION_INDEPENDENT_CODE.html